frontend/src/container/AppFullscreenRouter.jsx

```diff
@@ -36,79 +36,97 @@ export class AppFullscreenRouter extends React.Component {
 
     return (
       <div className='AppFullScreenManager'>
-        <Route exact path={PAGE.ADMIN.WORKSPACE} render={() => {
-          if (props.user.profile !== PROFILE.administrator.slug) return <Redirect to={{ pathname: '/ui' }} />
-
-          const content = {
-            workspaceList: [],
-            userList: []
-          }
-
-          props.renderAppFullscreen({ slug: 'admin_workspace_user', hexcolor: '#7d4e24', type: 'workspace' }, props.user, null, content)
-          return null
-        }} />
-
-        <Route exact path={PAGE.ADMIN.USER} render={() => {
-          if (props.user.profile !== PROFILE.administrator.slug) return <Redirect to={{ pathname: '/ui' }} />
-
-          const content = {
-            workspaceList: [],
-            userList: []
-          }
-
-          props.renderAppFullscreen({ slug: 'admin_workspace_user', hexcolor: '#7d4e24', type: 'user' }, props.user, null, content)
-          return null
-        }} />
-
-        <Route exact path={PAGE.AGENDA} render={() => {
-          const agendaConfig = {
-            workspaceId: null,
-            forceShowSidebar: true
-          }
-          props.renderAppFullscreen({ slug: 'agenda', hexcolor: '#7d4e24', appConfig: agendaConfig }, props.user, null, {})
-          return null
-        }} />
+        <Route
+          exact
+          path={PAGE.ADMIN.WORKSPACE} render={() => {
```

Comment:
this doesn't seem consistent to me.
Did a new eslint rule stage that the old syntax is wrong ?
About the new version, to be consistent, render props should be at a new line

Reply:
Yes: https://github.com/standard/eslint-config-standard-jsx/blob/master/eslintrc.json#L27
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-first-prop-new-line.md

But you are right, render should be at a new line and I will thix this

Reply:
And I just did.